client/src/config.tsx

```diff
@@ -322,8 +322,8 @@ export const appQuestions: Step[] = [
 ];
 
 export const TIMELINE: Deadlines = {
-  application: new Date('2025-02-28T00:00-08:00'),
-  decisions: new Date('2025-03-07T00:00-08:00'),
-  acceptance: new Date('2025-03-21T00:00-07:00'),
+  application: new Date('2025-03-09T00:00-08:00'),
+  decisions: new Date('2025-03-21T00:00-08:00'),
```

Comment:
because of DST the offset for this time should be `-07:00` 🙃 but it seems to be equivalent to 1 am here so it's not a big deal